### DISCUSSION TITLE
Is it possible to download and install an older version of a cask?

### DISCUSSION THREAD

**Discussion by JesseTG** [Homebrew/discussions#1274] [Casks] [answered]
Suppose I wanted to download and install an older version of Unity through its [cask](https://formulae.brew.sh/cask/unity). How can I do that? I'm having a hard time finding any kind of `-version` flag or something in the documentation.

**Comment by SMillerDev** [comment, marked as answer]
https://github.com/Homebrew/homebrew-cask-versions

**Reply by JesseTG** [reply]
Thank you, I appreciate it.

**Comment by marcobiedermann** [comment]
Same here. I would like to install an older version.
For example `Sketch@71.2`.
I've already tried `brew install --cask sketch@71.2` / `sketch@v71.2` \ `--version=71.2 sketch` but non or them worked.
`homebrew-cask-versions` seems to host versions like `alpha` or `beta` but not exact versions

Help is highly appreciated

**Reply by SMillerDev** [reply]
You can't install older versions, you could however make your own tap and host versions that download older software there.
